.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: Continous Integration
+
+on:
+  pull_request:
+    branches:
+      - master
+      - dev
+  push:
+    branches:
+      - master
+      - dev
+jobs:
+  unit-tests:
+    name: Unit Tests
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+
+      - name: Build unit tests project
+        run: |
+          mkdir build
+          cd build
+          cmake -DCMAKE_BUILD_TYPE=Release ..
+          cmake --build . --
```

Comment:
If you're not passing in any args to `build` (such as `-j12` etc.) you can omit the trailing `--`.